packages/xod-project/src/project.js

```diff
@@ -178,7 +181,35 @@ export const BUILT_IN_PATCHES = R.compose(
 // :: String -> Boolean
 export const isPathBuiltIn = R.flip(R.contains)(BUILT_IN_PATCH_PATHS);
 
-const getPatches = R.compose(R.merge(BUILT_IN_PATCHES), R.prop('patches'));
+const isConstructorPatch = R.compose(
+  R.any(R.equals(CONST.OUTPUT_SELF_PATH)),
+  R.map(Node.getNodeType),
+  Patch.listNodes
+);
+
+// :: Project -> Map PatchPath Patch
+const getPatches = memoizeOnlyLast(project => {
+  // TODO: better name.
```

Comment:
I'm bad in naming, but I think it's time to rename it and remove todo.

Reply:
> I'm bad at naming, but I think it's time to rename it and remove todo.

Yep, that TODO was more of "help me name it better", not "let's leave it like this and rename some time later"

> My proposals: original / genuine / real patches.

`genuine` sounds fine👍🏻